ivadomed/main.py

```diff
@@ -205,7 +244,9 @@ def cmd_train(context):
     if context['multichannel']:
         in_channel = len(context['multichannel'])
 
-    if context['retrain_model'] is None:
+    if context['unet_3D']:
+        model = models.Modified3DUNet(in_channels=in_channel, n_classes=1)
```

Comment:
The 2D-Unet is also not the original Unet --> what about calling it `Unet3D` or something like this and explain in the docstring the difference with the original 3D Unet?

Reply:
Would it be too much work to input the depth?

Reply:
Yes, will do!

Reply:
For the depth, I will investigate. It is probably possible!

Reply:
Reading briefly the article, the main differences are:
- instance normalisation
- "segmentation" layers on the decoder part of the Unet
- leaky ReLU
Is there something else?

Reply:
I think this sums it up :)!